iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java

```diff
@@ -22,39 +22,39 @@
 
     public static double getRatio(TwoWindingsTransformer twt) {
         double rho = twt.getRatedU2() / twt.getRatedU1();
-        if (twt.getRatioTapChanger() != null) {
+        if (twt.hasRatioTapChanger()) {
             rho *= twt.getRatioTapChanger().getCurrentStep().getRho();
         }
         return rho;
```

Comment:
You can refactor it:
`return twt.getRatedU2() / twt.getRatedU1() * twt.getOptionalRatioTapChanger().map(rtc -> rtc.getCurrentStep().getRho()).orElse(1);`